packages/expo-file-system/src/FileSystem.ts

```diff
@@ -69,6 +71,19 @@ export async function readAsStringAsync(
   return await ExponentFileSystem.readAsStringAsync(fileUri, options || {});
 }
 
+export async function getContentUriAsync(fileUri: string): Promise<string> {
+  if (Platform.OS === 'android') {
+    if (!ExponentFileSystem.getContentUriAsync) {
+      throw new UnavailabilityError('expo-file-system', 'getContentUriAsync');
+    }
+    return await ExponentFileSystem.getContentUriAsync(fileUri);
+  } else {
+    return new Promise(function(resolve, reject) {
```

Comment:
An interesting shorthand for the future would be `return Promise.resolve(valueToResolveWith)`, creating exactly what you did. 🙂

Reply:
Got it!